layer/CMakeLists.txt

```diff
@@ -50,26 +54,27 @@ else()
     set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wpointer-arith")
 endif()
 
-source_group("JSON Manifest" FILES VkLayer_khronos_profiles.json.in)
-source_group("Export Files" FILES VkLayer_khronos_profiles.def VkLayer_khronos_profiles.map)
+source_group("JSON Manifest" FILES ${LAYER_NAME}.json.in)
+source_group("Export Files" FILES ${LAYER_NAME}.def ${LAYER_NAME}.map)
 
-add_library(VkLayer_khronos_profiles MODULE)
-set_target_properties(VkLayer_khronos_profiles PROPERTIES FOLDER "Profiles layer")
+add_library(ProfilesLayer MODULE)
```

Comment:
why this change?

Reply:
Please consult with @juan-lunarg.

When doing a similar change to VVL, he recommended that instead of replacing the name of the layer through changing the CMake target, we should use the OUTPUT_NAME to change it.

Technically, the name of the CMake target could be left as is, but it really is historically named as such to match the output binary name, which is no longer relevant, hence a more generic name is probably preferred, just like on the VVL side the target name was changed from VkLayer_khronos_validation to simply vvl.

If there is a problem with that, I can change it, but I just tried to follow the same principle @juan-lunarg suggested on the VVL side.

Reply:
Ok, I am not seeing the point of that, the name of the binary is a pretty clear name for me for a target but ok.